src/lib/components/Select.svelte

```diff
@@ -7,10 +7,28 @@
 	export let options;
 	export let label;
 	export let name;
+
+	let isFocus = false;
+
+	function handleFocus() {
+		isFocus = true;
+	}
+
+	function handleBlur() {
+		isFocus = false;
+	}
 </script>
 
 <label class="relative flex">
-	<div class="text-[#DADADA] absolute top-1 left-4 text-xs">{label}</div>
+	<div
+		class="text-[#DADADA] absolute top-1 left-4 text-xs {selectedItem
```

Comment:
don't forget to make it transition
```suggestion
		class="text-[#DADADA] absolute transition-transform top-1 left-4 text-xs {selectedItem
```